### DISCUSSION TITLE
.net 5 ckeditor

### DISCUSSION THREAD

**Discussion by minhhungit** [serenity-is/Serenity#5433] [Help] [answered]
Hi team, I don't see ckeditor folder in wwwroot/Scripts, do we remove it 

![image](https://user-images.githubusercontent.com/2279508/106913289-6bd5ca00-6736-11eb-9312-0dd43508c610.png)

**Comment by VictorTomaili** [comment, marked as answer]
Check the serenity assets folder in serenity repository

**Reply by minhhungit** [reply]
@VictorTomaili  Just wonder will I copy the folder by manual ?
I installed Serenity.Assets package but still doesn't see the folder

I also ran `dotnet restore` and `dotnet sergen restore`

**Reply by VictorTomaili** [reply]
When you run the application in you pc, you already access these scripts from nugget package without doing anything. When you publish your app, you can see this scripts under your www folder. You don't need to do something. It's magic 😄

**Reply by minhhungit** [reply]
weird because I still got error with it

![image](https://user-images.githubusercontent.com/2279508/106917488-e4d72080-673a-11eb-8e96-2b53ccea401c.png)

I tried to create new project, it is .net 5 verson, Serene 5.0.24.0

**Reply by erehtolleh** [reply]
There are more details here in the migration notes.  I deleted a bunch of stuff and don't event know what I deleted when I upgraded from 3 to .NET 5. 
https://serenity.is/docs/migration/v5-0-12-to-v5-0-20

**Reply by minhhungit** [reply]
No @erehtolleh  I don't upgrade project from v3.1 to .net 5, I created **new** .net 5 web project

**Reply by VictorTomaili** [reply]
Yea, I see the problem. 

Serene package needs some fixes for that. 
Find `@Html.ScriptBundle("CKEditor")` 
replace CKEDITOR_BASEPATH on top of this line like this

 ```html
<script>var CKEDITOR_BASEPATH = '@ContentHashCache.ResolvePath(Context.Request.PathBase, "~/Serenity.Assets/Scripts/ckeditor/")';</script>
 ```
open `appsettings.bundles.json`
add `"~/Serenity.Assets/Scripts/ckeditor/skins/moono/editor.css"` in your site css bundles. Prefer before `site.css`

Probably you still has `split.js` issue. Manual copy this file to your www/scripts folder for just now. 

This is hotfix for now. I create an internal issue for this.  

Thanks for inform.

**Reply by minhhungit** [reply]
@VictorTomaili  thank you , it works
